.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: objective-c
-osx_image: xcode9
+osx_image: xcode10
 podfile: Example/Podfile
 
 before_install:
     - gem install cocoapods --pre --no-rdoc --no-ri --no-document --quiet
 
 script:
-    - set -o pipefail && xcodebuild test -project Marker/Marker.xcodeproj -scheme Marker-iOS -destination 'name=iPhone 7,OS=11.0' | xcpretty
+    - set -o pipefail && xcodebuild test -project Marker/Marker.xcodeproj -scheme Marker-iOS -destination 'name=iPhone 8,OS=12.0' | xcpretty
```

Comment:
You can specify the latest version of the OS instead of a specific version number, that might help the maintainability.

Reply:
I couldn't find the syntax for it